rest-api-spec/src/main/resources/rest-api-spec/api/indices.delete.json

```diff
@@ -20,6 +20,11 @@
         "master_timeout": {
           "type" : "time",
           "description" : "Specify timeout for connection to master"
+        },
+        "include_hidden": {
```

Comment:
they're not considered default params - if they're missing from the rest specs, they're not supported in the clients